dynatrace-aws-log-forwarder-template.yaml

```diff
@@ -518,6 +563,128 @@ Resources:
         - __Region__: !Ref 'AWS::Region'
           __LambdaName__: !Ref Lambda
 
+  EC2ActiveGate:
+    Condition: DeployVPCwithAG
+    Type: AWS::EC2::Instance
+    DependsOn: VPCPublicRouteInternet
+    Properties:
+      ImageId: !Ref LatestAmazonLinuxAmiId
+      InstanceType: "t3.small"
+      BlockDeviceMappings:
+        - DeviceName: "/dev/sdm"
+          Ebs:
+            VolumeType: "io1"
+            Iops: "200"
+            DeleteOnTermination: "true"
+            VolumeSize: "8"
+      SecurityGroupIds: [ !Ref VPCSecurityGroup ]
+      UserData:
+        Fn::Base64:
+          !Sub
+          - |
+            #!/bin/bash -xe
+            wget -O Dynatrace-ActiveGate-Linux-x86.sh "${DynatraceEnvironmentUrl}/api/v1/deployment/installer/gateway/unix/latest?arch=x86&flavor=default" --header="Authorization: Api-Token ${DynatracePaasToken}"
+            wget https://ca.dynatrace.com/dt-root.cert.pem ; ( echo 'Content-Type: multipart/signed; protocol="application/x-pkcs7-signature"; micalg="sha-256"; boundary="--SIGNED-INSTALLER"'; echo ; echo ; echo '----SIGNED-INSTALLER' ; cat Dynatrace-ActiveGate-Linux-x86.sh ) | openssl cms -verify -CAfile dt-root.cert.pem > /dev/null
```

Comment:
"-O name" just in case or not worth it?

Reply:
Not worth it, this instance is as clean as it gets :D

Reply:
Question, does this verification actually change/stop the flow? I guess it the one in the usual Deploy OneAgent page, and perhaps it can stay for debugging either way, just wondering.

Reply:
It should return non zero code and this will stop execution (-e bash option above). I will double check

Reply:
It returns non zero code (4) and this will stop the execution